### DISCUSSION TITLE
Possible more insights for the Yolo badge

### DISCUSSION THREAD

**Discussion by thomscoder** [Schweinepriester/github-profile-achievements#91] [General]
I'm sharing this discussion to keep an eye on: 
https://github.com/orgs/github-community/discussions/19885.
It might either be a bug or expand the ways to get the YOLO badge (self approving PR's)

**Comment by Schweinepriester** [comment]
Interesting, thank you!

Guess they really are working on the achievements still :D

**Reply by thomscoder** [reply]
yeah lol :)
